Redo Linux & Steamdeck install guide (#95)
goatcorp/faq: steamdeck.md

sh -c "$(curl -fsSL https://raw.githubusercontent.com/Blooym/xlm/main/setup/install-steamdeck.sh)"
```

**Steam Install Script (Native)**
```sh
sh -c "$(curl -fsSL https://raw.githubusercontent.com/Blooym/xlm/main/setup/install-native.sh)"
```

**Steam Install Script (Flatpak)**
```sh
sh -c "$(curl -fsSL https://raw.githubusercontent.com/Blooym/xlm/main/setup/install-flatpak.sh)"
```

After the installer has finished, please follow these steps to use the compatibility tool:
- Switch back to gaming mode (if on Steam Deck) or restart your Steam client otherwise.
- Navigate to your library and select "FINAL FANTASY XIV Online" or "FINAL FANTASY XIV Online Free Trial" if you are playing via the free trial or don't own the Steam edition of FFXIV. 
- Open the game properties menu and switch to the "compatibility" tab.
- Enable the "Force the use of a specific Steam Play compatibility tool" checkbox.
- From the box that appears select "XLCore [XLM]" (if this does not show, please make sure you properly restarted Steam).
- You can now launch the game as usual. XIVLauncher will be automatically installed and run for you.

## Install XIVLauncher as a flatpak or system package

### Flatpak

> First, check to see if flathub is installed with flatpak remotes. If you don't see an entry for flathub, check https://flathub.org/setup for instructions on how to install it.

Reviewer: ```suggestion
First, check to see if flathub is installed by running `flatpak remotes`. If you don't see an entry for flathub, check https://flathub.org/setup for instructions on how to install it.
```
flatpak remotes should be marked with back-tick/grave markers so it's obvious that it's a command to be run: `flatpak remotes`
